comms/dht/src/inbound/decryption.rs

```diff
@@ -193,8 +193,17 @@ where S: Service<DecryptedDhtMessage, Response = (), Error = PipelineError>
     ) -> Result<DecryptedDhtMessage, DecryptionError> {
         let dht_header = &message.dht_header;
 
+        let mut header_mac_bytes = dht_header.major.to_le_bytes().to_vec();
```

Comment:
minor perf: is it worth allocating this vec with capacity before? Maybe not necessary without a profile but there would be a few reallocations.

Reply:
I guess you might as well